src/common/lib/client/realtime.ts

```diff
@@ -14,17 +14,21 @@ import Platform from 'common/platform';
 import Message from '../types/message';
 
 class Realtime extends Rest {
-  channels: any;
```

Comment:
I'm not sure I understand why this change is needed here?

Reply:
tsc errors if you extend a class and override a dynamic accessor to be a static field, even if the return type of the accessor is the same as the static field